azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/model/AzureCredentialWrapper.java

```diff
@@ -16,8 +16,6 @@
 import java.util.List;
 import java.util.Objects;
 
-;
-
 public class AzureCredentialWrapper {
```

Comment:
use `@Nonnull`, so that the developer knows it doesn't accept null parameters
```
    public AzureCredentialWrapper(AuthMethod method, TokenCredential tokenCredential, AzureEnvironment env) {
        Objects.requireNonNull(method, "authMethod is null");
        Objects.requireNonNull(tokenCredential, "tokenCredential is null");
        Objects.requireNonNull(tokenCredential, "env is null");
```